engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EmailSubscriptionRepository.java

```diff
@@ -28,7 +28,7 @@ public List<String> getEmailSubscribersUserId(String accountId, String orgId, St
                     .setParameter("subscriptionType", subscriptionType)
                     .getResultList();
         } else {
-            String query = "SELECT es.id.userId FROM EmailSubscription es WHERE id.accountId = :accountId AND application.bundle.name = :bundleName " +
+            String query = "SELECT es.id.userId FROM EmailSubscription es WHERE accountId = :accountId AND application.bundle.name = :bundleName " +
```

Comment:
Are we keeping this branch?

Reply:
I'm guessing it will go away in a follow up cleanup :grin:

Reply:
Exactly :smiley: